lib/elixir/lib/range.ex

```diff
@@ -2,7 +2,7 @@ defmodule Range do
   @moduledoc """
   Defines a range.
 
-  A range represents a discrete number of integers.
+  A range represents a discrete number of consecutive integers.
```

Comment:
This is not true for ranges with the same start and end such as `1..1`

Reply:
Good catch, what do you think about #8473?